pkg/scheduler/controller.go

```diff
@@ -173,82 +187,55 @@ func (c *controller) startWorker(ctx context.Context) {
 
 func (c *controller) processNextWorkItem(ctx context.Context) bool {
 	// Wait until there is a new item in the working queue
-	k, quit := c.queue.Get()
+	i, quit := c.queue.Get()
 	if quit {
 		return false
 	}
-	key := k.(string)
+	item := i.(queueItem)
+	key := item.key
```

Comment:
As well as adding the rest of `item` to the logger, how about adding the object kind too?

Reply:
Makes sense, added in latest commits.